charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,187 @@
+
+  # Set a limit of memory that Tail plugin can use when appending data to the Engine.
+  # If the limit is reach, it will be paused; when the data is flushed it resumes.
+  # You can find more details here: https://docs.fluentbit.io/manual/pipeline/inputs/tail#config
+  # if the inbound traffic is less than 2.4Mbps, setting memBufLimit to 5MB is enough
+  # if the inbound traffic is less than 4.0Mbps, setting memBufLimit to 10MB is enough
+  # if the inbound traffic is less than 13.64Mbps, setting memBufLimit to 50MB is enough
+  input:
+    tail:
+      memBufLimit: 5MB
+    # uncomment below nodeExporterMetrics section if you want to collect node exporter metrics
+#   nodeExporterMetrics:
+#     tag: node_metrics
+#     scrapeInterval: 15s
+#     path:
+#       procfs: /host/proc
+#       sysfs: /host/sys
+
+  # Configure the output plugin parameter in FluentBit.
+  # You can set enable to true to output logs to the specified location.
+  output:
+#  You can find more supported output plugins here: https://github.com/fluent/fluent-operator/tree/master/docs/plugins/fluentbit/clusteroutput
+#    es:
+#      enable: false
+#      host: "<Elasticsearch url like elasticsearch-logging-data.kubesphere-logging-system.svc>"
+#      port: 9200
+#      logstashPrefix: ks-logstash-log
+#      path: ""
+#      bufferSize: "4KB"
+#      index: "fluent-bit"
+#      httpUser:
+#      httpPassword:
+#      logstashFormat: true
+#      replaceDots: false
+#      enableTLS: false
+#      tls:
+#        verify: On
+#        debug: 1
+#        caFile: "<Absolute path to CA certificate file>"
+#        caPath: "<Absolute path to scan for certificate files>"
+#        crtFile: "<Absolute path to private Key file>"
+#        keyFile: "<Absolute path to private Key file>"
+#        keyPassword:
+#        vhost: "<Hostname to be used for TLS SNI extension>"
+#    kafka:
+#      enable: false
+#      brokers: "<kafka broker list like xxx.xxx.xxx.xxx:9092,yyy.yyy.yyy.yyy:9092>"
+#      topics: ks-log
+#    stdout:
+#      enable: false
+
+  # Configure the default filters in FluentBit.It's our customized lua contained log format converter.
+  # The `filter` will filter and parse the collected log information and output the logs into a uniform format. You can choose whether to turn this on or not.
+#  filter:
+#    kubernetes:
+#      enable: true
+#      labels: false
+#      annotations: false
+#    containerd:
+#      enable: true
+#    systemd:
+#      enable: true
+
+fluentd:
+  enable: false
+  name: fluentd
+  port: 24224
+  image:
+    repository: "kubesphere/fluentd"
+    tag: "v1.14.6"
+  replicas: 1
+  forward:
+    port: 24224
+  watchedNamespaces:
+    - kube-system
+    - default
+  resources: {}
+  # limits:
+    # cpu: 100m
+    # memory: 128Mi
+  # requests:
+    # cpu: 100m
+    # memory: 128Mi
+  # Configure the output plugin parameter in Fluentd.
+  # You can set enable to true to output logs to the specified location.
+  output:
```

Comment:
Similar to the fluent bit one, what do we get by default?

Reply:
fluentd is optional and is not enabled by default, and if enabled fluentd, no default output will be enabled.
We'll add stdout output later because now we don't have an annotation field for fluentd crd

Reply:
OK, let's clarify that a little. I'll add a suggestion above.